qiime/core/type/primitive.py

```diff
@@ -349,6 +349,30 @@ def encode(self, value):
 Float = _Float('Float')
 
 
+class _Bool(_Primitive):
+    _valid_predicates = {Choices, Arguments}
```

Comment:
I don't think the `Choices` predicate makes sense. Anything but `Bool % Choices({True, False})` isn't worth parametrizing and that expression is identical to just `Bool`